source/frontend/src/features/mapSideBar/shared/FileMenuView.tsx

```diff
@@ -38,6 +48,39 @@ const FileMenuView: React.FunctionComponent<React.PropsWithChildren<IFileMenuPro
   // respect the order of properties as set by the user creating the file
   const sortedProperties = sortFileProperties(file?.fileProperties ?? []);
   const isSummary = useMemo(() => !exists(currentPropertyIndex), [currentPropertyIndex]);
+  const mapMachine = useMapStateMachine();
+
+  const fitBoundaries = () => {
+    const fileProperties = file.fileProperties;
```

Comment:
nit please change to `file?.fileProperties` to avoid null errors (like the rest of the code here)